english/choco-syrup/choco-syrup.md

```diff
@@ -0,0 +1,13 @@
+By all the marshmallow God's powers, we summoned together the raining of Hershey's chocolate
+
+You run across from end-to-end holding a rope with fire on it. Hoping to create a trail of 
+
+melted marshmallow that you can dip and eat with the Hershey's syrup rain.
+
+At this point, you have eaten all that you can. What are you going to do next?
+
+[Become a Chocomonster](be-chocomonster/chocomonster.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.